### DISCUSSION TITLE
Multiple input channels

### DISCUSSION THREAD

**Discussion by fgadaleta** [tomaka/cpal#1317] [Q&A] [answered]
Is there a way to set multiple input channels, e.g. mic and audio from a web call (or equivalent)?

**Comment by pietervandermeer** [comment, marked as answer]
You can select multiple devices for input and output. There's also support for multichannel, but Cpal prefers Stereo and Mono modes. You need to explicitly match an available configuration using a loop. Check the enumeration example.
